.gitattributes

```diff
@@ -0,0 +1,7 @@
+# This file is documented at https://git-scm.com/docs/gitattributes.
+# Linguist-specific attributes are documented at
+# https://github.com/github/linguist.
+
+**/zz_generated.*.go linguist-generated=true
+/pkg/clients/** linguist-generated=true
+/vendor/** linguist-generated=true
```

Comment:
The vendor tree is already covered by the linguist defaults:

https://github.com/github/linguist/blob/master/lib/linguist/vendor.yml#L112

Reply:
done